src/main/java/com/google/sps/servlets/utils/CommonWordsRetriever.java

```diff
@@ -40,4 +41,20 @@
   public static List<String> getCommonWords() {
     return new ArrayList<String>(commonWordsList);
   }
+
+  /**
+   * Check if the current environment is JUnit Test or not
+   *
+   * @return True if in Junit test; otherwise false
+   */
+  public static boolean isJUnitTest() {
```

Comment:
This will work, but for the record, this is not the best way to go about this.

Ideally, information about the "deployment" type of the code should come from some external centralizable config (e.g. from the maven config, a command line flag) since placing this information within the class makes it more complex.

I think the simplest approach would be to just directly configure the file path that should be used during testing and the file path that should be used during deployment. If you've expended a lot of effort looking into this already, then this approach could be okay, but ideally we wouldn't need to have this complicated logic.